src/Cli/dotnet/CommonOptions.cs

```diff
@@ -25,6 +25,9 @@ internal static class CommonOptions
         public static Option<VerbosityOptions> VerbosityOption() =>
             VerbosityOption(o => $"-verbosity:{o}");
 
+        public static Option<VerbosityOptions> HiddenVerbosityOption() =>
+            VerbosityOption(o => $"-verbosity:{o}").Hide();
```

Comment:
I'm curious, would this work?  Or would that end up hiding both options?

```suggestion
            VerbosityOption().Hide();
```

Reply:
That should work too!